### ISSUE TITLE
Enable deferred viewports for web

### ISSUE THREAD

**Issue by lukexor** [lukexor/tetanes#307]
#296 added support for deferred viewports for desktop platforms, but due to a limitation in winit 0.29.15, web is unsupported.

Once #289 is complete, remove the immediate viewport codepaths altogether.

**Comment by lukexor** [comment]
Fixed on https://github.com/lukexor/tetanes/pull/315
